docs/reference/pages/panels.rst

```diff
@@ -179,7 +179,7 @@ ImageChooserPanel
 
       class BookPage(Page):
           cover = models.ForeignKey(
-              'wagtailimages.Image',
+              'media.Image',
```

Comment:
It might be better to encourage use of `wagtail.images.get_image_model_string()` here instead